content/en/docs/setup/develop_with_kind.md

```diff
@@ -0,0 +1,124 @@
+---
+draft: false
+linktitle: Deploy/Develop with Kind
+menu:
+  docs:
+    parent: setup
+    weight: 4
+title: Deploy/Develop with Kind
+toc: true
+type: docs
+description: >
+  This guide provides an example for deploying and developing Kmesh.
+
+---
+# Kmesh Deploy and Develop in Kind
+
+[Kind](https://github.com/kubernetes-sigs/kind) is a convenient tool for quickly deploying kubernetes cluster locally. We can use `kind` to create an `istio` cluster and deploy `kmesh`.
+
+## Deploy Kmesh in Kind
+
+Let's start from setting up the required environment. You can follow the steps below:
+
++ Install `kind`:
+
+    Installing `kind` is very simple, because it's just a binary file. You can select the correct one according to the version and the architecture in the [github releases page](https://github.com/kubernetes-sigs/kind/releases). Take `linux` + `amd64` as example:
+
+    ```shell
+    wget -O kind https://github.com/kubernetes-sigs/kind/releases/download/v0.23.0/kind-linux-amd64
+    chmod +x kind
+    mv kind /usr/bin/
+    ```
+
++ Create Kubernetes cluster using `kind`:
+
+    You can take reference from the [istio official document](https://istio.io/latest/docs/setup/platform-setup/kind/).
+
+    If you want to specified multiple workers or node image, you can:
+
+    ```shell
+    kind create cluster --image=kindest/node:v1.23.17 --config=- <<EOF
+    kind: Cluster
+    apiVersion: kind.x-k8s.io/v1alpha4
+    name: ambient
+    nodes:
+    - role: control-plane
+    - role: worker
+    - role: worker
+    EOF
+    ```
++ Install `istioctl`:
+    ```shell
+    curl -L https://istio.io/downloadIstio | sh -
+    cd istio-1.22.2/bin
+    chmod +x istioctl
+    mv istioctl /usr/bin/
+    ```
++ Install istio components using `istioctl`
+    ```shell
+    istioctl install
+    ```
+    If you want to use `kmesh` in `workload` mode, you should deploy `istio` in [ambient mode](https://istio.io/latest/docs/ambient/overview/), by adding an extra flag:
+    ```shell
+    istioctl install --set profile=ambient 
+    ```
++ Install kubectl
+    Please follow the official guide: [Install and Set Up kubectl on Linux](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/).
++ Deploy Kmesh
+    Now, you are ready to deploy kmesh in your local cluster. Feel free to follow the [Kmesh Quick Start](https://kmesh.net/en/docs/setup/quickstart/).
+## Develop Kmesh in Kind
+You can follow the steps below to develop in kind:
++ Build your code locally:
+    ```shell
+    make build
+    ```
+    This will start a docker container named `kmesh-build` to build your code. 
++ Build your docker image locally:
+    ```shell
+    docker build --build-arg arch=amd64 -f build/docker/kmesh.dockerfile -t $image_name .
```

Comment:
Maybe we could tell user to use `make docker` to combine the above two steps in one ;) It seems more convenient.